stories/product-switch/product-switch.stories.js

```diff
@@ -34,7 +34,7 @@ export const Shellbar = () => `${localStyles}
     <div class="fd-product-switch">
         <div class="fd-popover fd-popover--right">
             <div class="fd-popover__control">
-                <button class="fd-button fd-button--transparent fd-popover__control fd-product-switch__control"
+                <button class="fd-button fd-button--transparent fd-popover__control"
```

Comment:
You have a div on line 36 with `fd-popover__control` class and the same class applied on the button. Maybe remove the parent div.